spec/forms/steps/income/income_before_tax_form_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Steps::Income::IncomeBeforeTaxForm do
+  subject(:form) { described_class.new(arguments) }
+
+  let(:arguments) do
+    {
+      crime_application:,
+      income_above_threshold:
+    }
+  end
+
+  let(:crime_application) { instance_double(CrimeApplication, applicant:) }
+  let(:applicant) { instance_double(Applicant, income_details:) }
+  let(:income_details) { instance_double(IncomeDetails) }
+
+  let(:income_above_threshold) { nil }
+
+  describe '#choices' do
+    it 'returns the possible choices' do
+      expect(
+        subject.choices
+      ).to eq([YesNoAnswer::YES, YesNoAnswer::NO])
```

Comment:
This is totally subjective, but I'm of the opinion in tests the 'expected' values should be hard-coded e.g. 'yes' to prevent circucar logic causing tests to pass but fail in production. It's really subjective tbh.

Reply:
I kinda get what you mean @mseedat-moj but I don't think this is circular in this case. Especially if that YesNoAnswer Type is tested. My understanding is that part of the value of using the YesNoAnswer is we could change the values to True False and this would all be fine. And it's the YesNoAnswer spec that is responsible for making sure the values are correct